onebusaway-android/src/main/java/org/onebusaway/android/ui/HomeActivity.java

```diff
@@ -357,6 +358,9 @@ public void onCreate(Bundle savedInstanceState) {
 
         mInitialStartup = Application.getPrefs().getBoolean(INITIAL_STARTUP, true);
 
+        // Workaround to make sure ES SDK is initialized in case we startup to ES Fragments (#953)
+        Application.get().setUpSocial();
```

Comment:
This is too late.  setUpSocial should be called before setContentView (line 355) since that calls the NavigationDrawerFragment's onCreate() which forces it to select the last saved item (in this case the ES fragment)